app/controllers/bookings_controller.rb

```diff
@@ -83,8 +87,22 @@ def examine_booking(response)
       end
     end
 
+    def is_ticket_quantity_specified
```

Comment:
Inconsistent indentation detected.<br>Rename `is_ticket_quantity_specified` to `ticket_quantity_specified?`.